basic-wallet/src/App.tsx

```diff
@@ -0,0 +1,95 @@
+
+import React, { useState, useEffect } from 'react'
+
+import * as serviceWorker from './serviceWorker';
+import { xdai, eth } from '@burner-wallet/assets';
+import BurnerCore from '@burner-wallet/core';
+import { InjectedSigner, LocalSigner } from '@burner-wallet/core/signers';
+import { InfuraGateway, InjectedGateway, XDaiGateway, } from '@burner-wallet/core/gateways';
+import Exchange, { Uniswap, XDaiBridge } from '@burner-wallet/exchange';
```

Comment:
We may need to look at the xdai gateway, xdai bridge, and uniswap libraries and make sure we're using the idchain equivalents, e.g. https://uniswap.idchain.one https://dai.idchain.one